survey/summary.md

```diff
@@ -0,0 +1,130 @@
+the most supported signature type), with some a couple ecosystems with no
+support for signing yet - but there is interest/plans.
+
+There are no package managers that support keyless signing yet.
+
+Java, OCaml, Go, Ruby and Rust support reproducible builds with Maven Central
+making it mandatory.
+
+![image](https://user-images.githubusercontent.com/3060102/220680128-180c076c-31de-4f32-884e-7c2de8137dc0.png)
+
+## Typos & dependency confusion
+
+45.5% of package managers do not require and do not plan on having DNS
+verification proof when reserving/exercising a domain/namespace or domain name
+attribute of a package. This could lead to impersonation of
+organisations/entities. A method to reduce the cost of protecting against this
+threat or education of the seriousness of this threat is needed.
+
+General typos and dependency confusion is observed and handled in a myriad of
+different ways across ecosystems. It would be helpful to provide guidance on a
+list of methods and evaluation of their effectiveness.
+
+![image](https://user-images.githubusercontent.com/3060102/220680273-500cc331-728b-48e6-a8e6-8c5fd131b57a.png)
+
+## Handling compromise and malice
+
+Most ecosystems don't see the responsibility of finding malware in the packages
+as their responsibility. However, they provide the ability to flag and handle
+malware in releases when it happens.
+
+Rust, Ruby and Gradle have no plans on ensuring that install and build code
+doesn't allow execution of arbitrary code on the host in an unprotected way.
+This is an area that requires further investigation into why that is not a
+priority and look into a way to be able to have isolation and provenance of
+builds without this property.
+
+![image](https://user-images.githubusercontent.com/3060102/220681554-3b469808-3fdd-441e-b91c-0cf060461af2.png)
+
+## Protecting the ecosystem
+
+Most ecosystems are not currently performing any security testing against its
+code/infrastructure. However, all of them are interested (7) or have it planned
+(2). These ecosystems could be funded for security testing.
+
+All ecosystems have a process in place for vulnerability disclosure or are
+interested in doing so.
+
+Only 1 ecosystem has additional supply chain security guarantees for the
+binaries/tooling that it produces. This is an area for improvement overall.
+
+![image](https://user-images.githubusercontent.com/3060102/220682148-2d488135-4695-41b5-a6ab-e5f85883f612.png)
+
+## Auth and Credential Management
+
+Generally there is interest in protecting token security, which is positive
+feedback. There are 2 ecosystems that currently have no plans on requiring MFA.
+Further investigation on why would be helpful.
+
+## Policy:
+
+With policy, there is a mix of implementations that help provide ability to
+create policy on keys/hashes as well as checking for known vulnerabilities, as
+well as a significant number of ecosystems (4-5) which do not but are
+interested.
```

Comment:
I think splitting this into 2, perhaps 3, sentences would make it easier to read.